obspy/signal/src/pk_mbaer.c

```diff
@@ -1,17 +1,20 @@
 /*--------------------------------------------------------------------
 # Filename: pk_mbaer.c
-#  Purpose: C-Verions of Baer Picker based on Fortran code of Baer
+#  Purpose: C-Version of Baer Picker based on Fortran code of Baer
 #   Author: Andreas Rietbrock, Joachim Wassermann
 # Copyright (C) A. Rietbrock, J. Wassermann
+#
+# Modified: M.Bagagli 02-2019
+            - The function now returns also the characteristic function
```

Comment:
This is not necessary. `git` keeps track of all changes an the reasoning behind it.